File: backend/pkg/errcode/errcode.go
// 错误码定义
package errcode

const (
	DBERR_BASE  string = "1000"
	DB_CONN_ERR string = "1001"
	DB_NO_ROWS  string = "1002"
	DB_DUP_ERR  string = "1062"

	SERVER_BASE    string = "2000"
	MISSING_PARAM  string = "2001"
	WRONG_PASSWORD string = "2002"
	JWT_ERR        string = "2003"
	AUTH_ERR       string = "2004"
	WRONG_PARAM    string = "2005"

	MQ_BASE     string = "3000"
	MQ_CHAN_ERR string = "3001"
)

File: frontend/src/utils/errcode.js
// 错误码常量
export const errcode = {
  // 数据库相关错误
  DB_CONN_ERR: '1001',  // 数据库连接错误
  DB_DUP_ERR: '1062',   // 数据重复错误
  
  // JWT相关错误
  JWT_ERR: '2001',      // JWT错误
  
  // 参数相关错误
  WRONG_PARAM: '3001',  // 参数错误
  
  // 权限相关错误
  NO_PERMISSION: '4001' // 权限不足
} 